docker/helm-api-subscribe-submit-test

```diff
@@ -1,75 +1,108 @@
 #!/usr/bin/env sh
 set -e
-
+# shellcheck disable=SC2153
+jwt="$TOKEN"
 echo "HOST: $HOST"
 echo "PORT: $PORT"
+if [ "$REQUIRE_AUTH" = "true" ]; then
+  echo "REQUIRE_AUTH: $REQUIRE_AUTH"
+else
+  echo "Testing without authorization header ..."
+fi
 
-TIMEOUT=60
+TIMEOUT=0
```

Comment:
We know this works reliably, so waiting with no timeout means you could `helm test` at startup.